packages/components/forms/src/radio/Radio.mdx

```diff
@@ -170,6 +172,8 @@ The Radio can be rendered as disabled with the `isDisabled` property.
 ## Accessibility
 
 - To ensure `Radio` meets accessibility standards, provide `name` and `id`
+- When using `Radio.Group` it whould be wrapper by a `fieldset` and have a `legend`.
+  - We also provide `FormControl` and `FormControl.Label` that can be rendered as `fieldset` and `legend`
```

Comment:
Same here, maybe we could merge those 2 points, they seem to talk about the same thing? or I get it wrongly?